### PR TITLE
Made Histograms sheet optional.

### PR DESCRIPTION
**Pull request by JS3xton**
See #182.

### CODE REVIEW

**Comment by JS3xton** [comment]
Under `develop` (c51a9394e2ca48391e6fb9df965f58e6913b6cf0),
```
>python -m FlowCal.excel_ui -i examples/experiment.xlsx -o dev_out.xlsx
```
yielded:
dev_out

Under 3bf8a304544a9dfa817908727a5249e48e1699b6,
```
>python -m FlowCal.excel_ui -i examples/experiment.xlsx -o new_out.xlsx
```
yielded:
new_out

Under 3bf8a304544a9dfa817908727a5249e48e1699b6,
```
>python -m FlowCal.excel_ui -i examples/experiment.xlsx -o new_out_H.xlsx -H
```
yielded:
new_out_h

File: FlowCal/excel_ui.py [hunk 1]
```diff
@@ -1324,11 +1324,11 @@ def run(input_path=None, output_path=None, verbose=True, plot=True):
...
      5. Process all the cell samples in the Samples table.
      6. Generate statistics for each sample.
-     7. Generate a histogram table for each fluorescent channel specified
-        for each sample.
+     7. If specified, generate a histogram table for each fluorescent channel
+        specified for each sample.
      8. Generate a table with run time, date, FlowCal version, among
         others.
-     9. Save statistics and histograms in an output Excel file.
+     9. Save statistics and (if specified) histograms in an output Excel file.
```
Comment (JS3xton) [inline FlowCal/excel_ui.py]: "requested" might be a better word than "specified" in this description, but this is minor

Comment (castillohair) [reply FlowCal/excel_ui.py]: Yeah, requested sounds better.

**Comment by castillohair** [comment]
I think we should include the -H flag in both "Run FlowCal" batch files. Comments on this?

**Comment by BrianLandry** [comment]
We definitely should, since the advertised workflow in the paper is running the "Run FlowCal" batch file and the paper says that that workflow will generate histograms.

Thanks for doing this @JS3xton

**Comment by JS3xton** [comment]
Oh shoot, I forgot about the batch files; omitting the `-H` flag there was not meant to advocate a position on that issue.

Yeah I agree with you guys; the out-of-the-box workflow should generate histograms since that's how it's described in the paper.

**Comment by castillohair** [comment]
Batch files work with 299e13e4e2bfad77700f0169136d2ab2b24c3691. Will merge now.

**Pull request merged**